source/PhysicalQuantity/Volume.php

```diff
@@ -129,5 +129,43 @@ protected static function initialize()
         $newUnit->addAlias('cup');
         $newUnit->addAlias('cups');
         static::addUnit($newUnit);
+
+		// teaspoon
+		$newUnit = UnitOfMeasure::linearUnitFactory('tsp', 0.00000492892);
+		$newUnit->addAlias('teaspoon');
+		$newUnit->addAlias('teaspoons');
+		static::addUnit($newUnit);
+
+		// tablespoon
+		$newUnit = UnitOfMeasure::linearUnitFactory('tbs', 0.00001478676);
+		$newUnit->addAlias('tablespoon');
+		$newUnit->addAlias('tablespoons');
+		static::addUnit($newUnit);
+
+		// Gallon
+		$newUnit = UnitOfMeasure::linearUnitFactory('gallon', 3.7854118e-3);
```

Comment:
Got it, good to know about that standard, I had seen some used the abbreviation, and some used the full word, I'll make those changes accordingly.